Assignment/solution_student/css/style.css

```diff
@@ -1,27 +1,184 @@
 /*
-Project Name: HTML Template
-Client: Your Client
-Author: Your Name
-Developer: @poopsplat in Chicago
+Project Name: Relaxr Landing Page
+Client: Relaxr
+Author: Andrea Watts
+Developer: @wattsand in Chicago
 */
 
 /******************************************
+/* SETUP
+/*******************************************/
+/* Box Model Hack: http://www.paulirish.com/2012/box-sizing-border-box-ftw/ */
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+}
+/* Micro clearfix hack: http://nicolasgallagher.com/micro-clearfix-hack/ */
+.clearfix {
+  *zoom: 1;
+}
+
+.clearfix:before,
+.clearfix:after {
+    content: " ";
+    display: table;
+}
+
+.clearfix:after {
+  clear: both;
+}
+
+/******************************************
 /* BASE STYLES
 /*******************************************/
+
 body {
   color: black;
   font-size: 16px;
-  background: white;
+  font-family: 'open sans', sans-serif;
+  width: 50%;
+  margin: 0 auto;
+  opacity: .85;
 }
 
-/******************************************
-/* LAYOUT
-/*******************************************/
+header{
+	color: #606161;
+	background-image: url(../images/header_bg.jpg);
+	background-position: center top;
+	background-size: cover;
+	width: 100%;
+	height: 375px;
+}
 
-/******************************************
-/* PAGES
-/*******************************************/
+h1,h2{
+	color: #f0efef;
+	font-size: 24px;
+	letter-spacing: 1px;
+}
 
-/******************************************
-/* SECTIONS
-/*******************************************/
\ No newline at end of file
+h1{
+	font-weight: 800;
+	padding-top: 10px;
+	padding-left: 50px;
+}
+
+h2{
+	font-weight: 600;
+	padding-top: 40px;
+}
+
+h3{
+	color: #033048;
+	font-size: 20px;
+	font-weight: 600;
+}
+
+h4{
+	color: #f9e42e;
+	font-size: 24px;
+	font-weight: 800;
+	margin: 0;
+	padding-top: 10px;
+	padding-bottom: 25px;
+}
+
+a{
+	display: inline-block;
+	padding-right: 10px;
+	padding-left: 10px;
+}
+
+h6{
+	color: #f0efef;
+	font-size: 9px;
+	font-weight: 600;
+}
+
+
+blockquote{
+	color: #f0efef;
+	font-size: 18px;
+	font-weight: 300;
+	font-style: italic;
+  	display:block;
+  	position: relative;
+  	background-image: url(../images/testimonial_bg.jpg);
+	background-position: center top;
+	background-size: cover;
+	width: 100%;
+	height: 200px;
```

Comment:
I would advise to _not_ specify an explicit height value if you can avoid it, and let the height of the content dictate the height of the container. This way, if the content changes, or the size of the screen viewing the site changes, it adjusts accordingly. In your case, after you add the bottom padding I suggest next, all you need to do is remove the height and you're good to go! See this as an example: [http://codepen.io/poopsplat/pen/PZzEPm](http://codepen.io/poopsplat/pen/PZzEPm)